catalogs/catalog-kafka/src/main/java/com/datastrato/gravitino/catalog/kafka/KafkaCatalogOperations.java

```diff
@@ -181,6 +184,21 @@ public NameIdentifier[] listTopics(Namespace namespace) throws NoSuchSchemaExcep
     }
   }
 
+  @Override
+  public void testConnection(
+      NameIdentifier catalogIdent,
+      Catalog.Type type,
+      String provider,
+      String comment,
+      Map<String, String> properties) {
+    try {
+      adminClient.listTopics().names().get();
```

Comment:
Why do you use an already-created `adminClient` to test Connection since you intend to test connection with the parameter?

Reply:
The `adminClient` was initialized with the provided parameters